addons/website/models/ir_http.py

```diff
@@ -289,7 +272,7 @@ def _serve_page(cls):
                 page.cache_time  # cache > 0
                 and request.httprequest.method == "GET"
                 and request.env.user._is_public()    # only cache for unlogged user
-                and 'nocache' not in request.params  # allow bypass cache / debug
+                and 'nocache' not in request.httprequest.args  # allow bypass cache / debug
```

Comment:
`nocache` in only set in unittests and via the query-string, we could had used `get_http_params()` but it is faster to get it from the qs right away.

Reply:
in 99% of case we use request.param and request.get_http_params, it is only to get the args and not the join of files, form, ...
So if we consider that we should use httprequest.args in this case, it bring a 3rd black sheep :sheep: :)

Reply:
Yes you are right, actually I tried to use `httprequest.args` everywhere except where I didn't know for sure the option was only sent via the query-strings. Then during the review people asked me to use `get_http_params()` everywhere. I think I forgot about this one. Monday we had a talk with rco who thinks we should use `httprequest.args` more... So you have it, a mix of `params`, `get_http_params` and this `httprequrest.args` that I forgot. And still discussions about whether we should use which one.

You are right, to be consistent this should had been `get_http_params`, I'm not changing it because we are actively discussing the mater. This line *will* change in the near future.